src/backend/marsha/core/api.py

```diff
@@ -130,10 +130,20 @@ def get_permissions(self):
             permission_classes = [IsAuthenticated]
         elif self.action in ["retrieve"]:
             permission_classes = [
-                permissions.IsPlaylistAdmin | permissions.IsPlaylistOrganizationAdmin
+                permissions.IsPlaylistAdmin
+                | permissions.IsPlaylistOrganizationAdmin
+                | (
```

Comment:
can you add a comment for each permission to clarify who you target?

Reply:
May I ask you why I should add a comment ?
Isn't permissions docstrings enough ? Or maybe I'm doing something wrong here, and you're wondering why I added those ?